Bump up version

syohex opened the pull request.

zonuexe commented:
Sorry, my suggestion (https://github.com/ejmr/php-mode/pull/329#issuecomment-275590549) was to release the condition **before** MR was merged (https://github.com/ejmr/php-mode/commit/a6c998937341f49138f07c15050efe7e5809be23) as stable version (for [MELPA stable](https://stable.melpa.org/#/php-mode)).

syohex commented:
I see. However it is about time bumping version, because we added some breaking backward compatibility changes since last version.

If we find many problems in latest version, then I consider about tag(Revert commits and tag again etc).
